Index: gdm-3.15.92/data/gdm.conf-custom.in
===================================================================
--- gdm-3.15.92.orig/data/gdm.conf-custom.in
+++ gdm-3.15.92/data/gdm.conf-custom.in
@@ -4,15 +4,31 @@
 # Uncoment the line below to force the login screen to use Xorg
 #WaylandEnable=false
 
+# Enabling automatic login
+#  AutomaticLoginEnable = true
+#  AutomaticLogin = user1
+
+# Enabling timed login
+#  TimedLoginEnable = true
+#  TimedLogin = user1
+#  TimedLoginDelay = 10
+
+# Reserving more VTs for test consoles (default is 7)
+#  FirstVT = 9
+
 [security]
 
 [xdmcp]
 
 [greeter]
+# Only include selected logins in the greeter
+# IncludeAll = false
+# Include = user1,user2
 
 [chooser]
 
 [debug]
 # Uncomment the line below to turn on debugging
+# Additionally lets the X server dump core if it crashes
 #Enable=true
 
